### DISCUSSION TITLE
Markdown-Oxide is now avaliable on Mason.nvim

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#32] [Announcements]
As of https://github.com/mason-org/mason-registry/pull/4683, Markdown oxide is now available in Mason.nvim. 

Support for mason-lspconfig is still to come...

**Comment by Feel-ix-343** [comment]
Mason-lspconfig PR [here](https://github.com/williamboman/mason-lspconfig.nvim/pull/378)

**Reply by Feel-ix-343** [reply]
merged!

**Comment by anandkumar89** [comment]
Issues with installation. Are there any dependencies required? I tried installing from Mason but caught up in error.

```
[INFO  Fri Mar 29 14:27:05 2024] ...vim/plugged/mason.nvim/lua/mason-core/installer/init.lua:184: Executing installer for Package(name=markdown-oxide) {}
[ERROR Fri Mar 29 14:27:05 2024] ...vim/plugged/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=markdown-oxide) error=spawn: cargo failed with exit code - and signal -. cargo is not executable
```

Edit : realised it depends on rust and rust is required to build It locally. Could it be possible in future to have a standalone packaged release?

**Reply by Feel-ix-343** [reply]
Hey!

I am actually working on releasing this today. It will take a few days to be merged into mason however

**Reply by anandkumar89** [reply]
noticed the binary release on repo. Will wait till it's integrated with mason. thanks :)

**Reply by Feel-ix-343** [reply]
It's in!

**Reply by anandkumar89** [reply]
Wow! let me test it!!
